contracts/suilend/sources/obligation.move

```diff
@@ -1221,4 +1370,57 @@ module suilend::obligation {
             user_reward_manager_index
         }
     }
-}
+
+    fun get_ltvs<P>(
+        obligation: &Obligation<P>,
+        deposit_reserve: &Reserve<P>,
+        is_emode: bool,
+    ): (Decimal, Decimal) {
+        if (is_emode) {
+            // check_normal_ltv_against_emode_ltvs
+            let borrow_reserve_index = get_single_borrow_array_reserve_if_any(obligation);
+
+            assert!(
```

Comment:
we never want this assert to get tripped because then obligation::refresh can abort, which would be very bad.

Reply:
in this case we still want to return the base ltvs, right?